backend_new/src/services/app.service.ts

```diff
@@ -0,0 +1,11 @@
+import { Injectable } from '@nestjs/common';
+import { SuccessDTO } from '../dtos/shared/success.dto';
+
+@Injectable()
+export class AppService {
+  heartbeat(): SuccessDTO {
+    return {
+      success: true,
```

Comment:
at some point it would be ideal to have the heartbeat endpoint also make sure that the db is accessible. I wonder if we should just do it now?

Reply:
Nice call out! I got this updated so we do the most simple call possible, lemme know what you think